src/main/java/decodes/tsdb/TsdbAppTemplate.java

```diff
@@ -244,7 +248,12 @@ public void execute(String args[])
 			}
 			catch(BadConnectException ex)
 			{
+				JOptionPane.showMessageDialog(null,ex.getLocalizedMessage());
```

Comment:
If this is running as service , the dialog may cause an issue?

Reply:
Oh yeah, TsdbAppTemplate isn't gui only, I should check before popping that open. I'm not super worried about the current usage though.

Reply:
I'll be going through and cleaning this up after the current release cycle.

If I'm lucky this might actually be the path forward on properly "merging" the DECODES and Time Series database within the code.